### PR TITLE
net-ftp/uftpd: fix failing test phase

### PR DESCRIPTION
**Pull request by oz123**
Bug: https://bugs.gentoo.org/733782
Package-Manager: Portage-2.3.103, Repoman-2.3.23
Signed-off-by: Oz Tiram <oz.tiram@gmail.com>

### CODE REVIEW

File: net-ftp/uftpd/uftpd-2.13-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	>=dev-libs/libite-1.5
+	>=dev-libs/libuev-2.2"
+
+DEPEND="
+	${RDEPEND}
+	!net-misc/uftp
+	!net-ftp/atftp
+	test? (
+		net-ftp/ftp
+		net-ftp/tnftp )"
```
Comment (thesamesam) [inline net-ftp/uftpd/uftpd-2.13-r1.ebuild:25]: Indentation (move the ) onto a new line)

Comment (oz123) [reply net-ftp/uftpd/uftpd-2.13-r1.ebuild]: Fixed.

**Comment by oz123** [comment]
@thesamesam as requested rev bump was removed and I added the missing dependency.
Thanks for reviewing.

**Comment by thesamesam** [comment]
Hmmmm:
```
 * Messages for package net-ftp/uftpd-2.13:

 * This package will overwrite one or more files that may belong to other
 * packages (see list below). You can use a command such as `portageq
 * owners / <filename>` to identify the installed package that owns a
 * file. If portageq reports that only one package owns a file then do
 * NOT file a bug report. A bug report is only useful if it identifies at
 * least two or more packages that are known to install the same file(s).
 * If a collision occurs and you can not explain where the file came from
 * then you should simply ignore the collision since there is not enough
 * information to determine if a real problem exists. Please do NOT file
 * a bug report at https://bugs.gentoo.org/ unless you report exactly
 * which two packages install the same file(s). See
 * https://wiki.gentoo.org/wiki/Knowledge_Base:Blockers for tips on how
 * to solve the problem. And once again, please do NOT file a bug report
 * unless you have completely understood the above message.
 *
 * Detected file collision(s):
 *
 *      /usr/sbin/in.tftpd
 *      /usr/share/man/man8/in.tftpd.8.bz2
 *
 * Searching all installed packages for file collisions...
 *
 * Press Ctrl-C to Stop
 *
 * net-ftp/tftp-hpa-5.2-r1:0::gentoo
 *      /usr/sbin/in.tftpd
 *      /usr/share/man/man8/in.tftpd.8.bz2
 *
 * Package 'net-ftp/uftpd-2.13' NOT merged due to file collisions. If
 * necessary, refer to your elog messages for the whole content of the
 * above message.
```

Now, this is odd, because obviously you've added it as a test dependency. Is something else providing tftp on your system?

**Comment by oz123** [comment]
@thesamesam well, I did something which isn't kosher apparently. I ran ebuild test directly, without thinking that emerge would do something else. The reason for that I added tftp-hpa is that in debian the package is split into two: the client and the server. I wasn't aware of that. 
Splitting the package in the debian way is probably something that's not considerable right now.
I see two options:
 - add dependency on busybox which supplies some kind of tftp client
 - disabling the test that requires tftp

Which way do you think is better?

**Comment by thesamesam** [comment]
Uhh.
```
$ emerge -v1 --with-test-deps uftpd

These are the packages that would be merged, in order:

Calculating dependencies... done!
[ebuild  N     ] dev-libs/libite-2.2.0::gentoo  USE="-static-libs" 261 KiB
[ebuild  N     ] dev-libs/libuev-2.3.1::gentoo  USE="-static-libs" 242 KiB
[ebuild  N     ] net-ftp/tnftp-20151004::gentoo  USE="ipv6 -socks5" 600 KiB
[ebuild  N     ] net-ftp/tftp-hpa-5.2-r2::gentoo  USE="client ipv6 readline server (-selinux) -tcpd" 88 KiB
[ebuild  N     ] net-ftp/ftp-0.17.23.0.2.1::gentoo  USE="ipv6 readline ssl -libressl" 104 KiB
[ebuild  N     ] net-ftp/uftpd-2.13::gentoo  USE="-test" 99 KiB
[blocks B      ] net-ftp/uftpd ("net-ftp/uftpd" is blocking net-ftp/tftp-hpa-5.2-r2)

Total: 6 packages (6 new), Size of downloads: 1,392 KiB
Conflict: 1 block (1 unsatisfied)

 * Error: The above package list contains packages which cannot be
 * installed at the same time on the same system.

  (net-ftp/uftpd-2.13:0/0::gentoo, ebuild scheduled for merge) pulled in by
    uftpd
```

**Comment by oz123** [comment]
Hi @thesamesam , yes this is not done yet. tftpd-ha was split, but it has collisions with other ftp servers in gentoo. I also need to twick the dependencies for uftpd here. 
Thanks for taking another look (and for ` --with-test-deps`. I didn't know this flag).

**Pull request closed without merging**